reference/entry.json

```diff
@@ -670,6 +670,129 @@
         ]
       }
     },
+    "/events/{eventId}/videos": {
+      "parameters": [
+        {
+          "$ref": "#/components/parameters/eventId"
+        }
+      ],
+      "post": {
+        "summary": "Add a video to an event",
```

Comment:
Nit: Other operations don't contain something like `to an event` or `on an event`. It's pretty clear from the context in the menu.

<img width="292" alt="Screen Shot 2021-10-05 at 11 24 28" src="https://user-images.githubusercontent.com/959026/135996874-1dd12248-99d2-49ca-8d51-94570d9a1d22.png">

(Same for the place operation)

Reply:
Oh yes, makes sense to remove the entity